doc/build-unix.md

```diff
@@ -338,7 +340,7 @@ Then build using:
 
     ./autogen.sh
     ./configure --with-incompatible-bdb BDB_CFLAGS="-I/usr/local/include/db5" BDB_LIBS="-L/usr/local/lib -ldb_cxx-5"
-    make
+    gmake
```

Comment:
I know, I know... 😆  It's easy to guess due to the error output of `make`, but... Anyway, thanks for docs. It was short and easy.